src/sentry/static/sentry/app/views/settings/components/settingsNavigation.tsx

```diff
@@ -44,14 +45,27 @@ class SettingsNavigation extends React.Component<Props> {
     const navWithHooks = navigationObjects.concat(hookConfigs);
 
     return (
-      <div>
+      <PositionStickyWrapper>
         {navWithHooks.map(config => (
           <SettingsNavigationGroup key={config.name} {...otherProps} {...config} />
         ))}
         {hooks.map((Hook, i) => React.cloneElement(Hook, {key: `hook-${i}`}))}
-      </div>
+      </PositionStickyWrapper>
     );
   }
 }
 
+const PositionStickyWrapper = styled('div')`
+  position: sticky;
+  top: 100px;
+  overflow: scroll;
+  height: calc(100vh - 98px);
```

Comment:
Is 98 px how tall the header above the sidebar is?

Reply:
How tall the header is, plus the padding above the nav.